### ISSUE TITLE
servoshell: Visual order and focus order of search input and experimental prefs button don't match

### ISSUE THREAD

**Issue by lukewarlow** [servo/servo#40926]
**Describe the bug:**
When using keyboard navigation in the servoshell UI you reach the experimental preferences button before the search input, despite it being rendered after the input.

**To Reproduce:**
Load Servo, press Tab key 4 times and you should be in the address bar, but you're actually on the toggle button.

**Platform:**
macOS but presumably all.

**Comment by Keerti707** [comment]
Hi,I would like to work on this issue!
Please assign it to me.
